### ISSUE TITLE
Reimplementation details

### ISSUE THREAD

**Issue by Xinjie-Q** [QiZhao-NJU/Difference-NeRV#1]
Hello! Great Work! 
Could you release the related training code for DNeRV? I have tried to use E-NeRV code to train DNeRV with UVG dataset. However, the reimplementation results differ significantly from those reported in the paper. For example, with 3M decoder, 2x40x80 difference embedding and L1+SSIM loss, the reimplemented beauty result was 33.83dB, while the reported beauty result was 40.00dB. The performance gap is so large, thus I sincerely hope you can provide more training details to help me get correct results.  
Thanks a lot and looking forward to your reply!

**Comment by QiZhao-NJU** [comment]
Thanks for your attention and comments to our work!
It should be noted that, all the NeRV methods would encounter the problem of training failure, where PSNR does not continuously rise, but suddenly drops in a certain training epoch. Whenever this situation occurs, the training efficiency of nerv will be affected. To solve this problem, we can replace the random seed. Also, the kernel size in CCU is changed as 3x3 for better performance.

**Comment by Aaronbtb** [comment]
Thank you for your work. I reviewed your article and found a similar problem.
I used your latest code and the training was set to random seed. With 3M decoder, 2x40x80 difference embedding and L2 loss, the reimplemented beauty result was 33.91dB. However, this is still far from the 40dB target. I would like to ask you for your advice on this model training problem.
Thank you very much for your help.

**Comment by jinsingsangsung** [comment]
I also failed to achieve 40~ish PSNR on Beauty. Mine was 34.18, so I'd say it's a bit better than the others here. Could you provide the exact seed you used to obtain the performance listed in the paper?

**Comment by QiZhao-NJU** [comment]
We will update the results on UVG later, as some of the UVG data we used in previous experiments had some problems, which were discovered recently. In addition, training instability and crashes will also affect the experimental results.Thank you for your attention to our work.
